content/course-specific-material/introR-NatGW-Aug16.md

```diff
@@ -1,11 +1,15 @@
 ---
+author: 
 date: 2016-08-29
 slug: introR-NatGW-Aug16
+draft: True
 title: National GW Workshop
-menu: 
-  main:
-    parent: Course Specific Material
-    weight: 1
+categories: Data Science
+ 
+tags: 
+  - R
+ 
+ 
```

Comment:
The `USGSmarkdowntemplates` is streamlined for the blogs. If updating the training becomes a fairly common task, we can either fiddle with that package, or make a new fork of it for the training stuff.